Specs/DateTime/Japanese/TimeExtractor.json

```diff
@@ -456,33 +421,30 @@
   },
   {
     "Input": "真夜中",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "真夜中",
+        "Text": "夜中",
         "Type": "time",
-        "Start": 0,
-        "Length": 3
+        "Start": 1,
+        "Length": 2
       }
     ]
   },
   {
-    "Input": "午前中",
-    "NotSupported": "dotnet",
+    "Input": "午前半ば",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "午前中",
```

Comment:
Do we have this case in the time range specs?

Reply:
yes, there are a couple of cases already